experiments/simple_train_config.py

```diff
@@ -68,6 +68,12 @@ class SimpleTrainConfig:
     optimizer_config: OptimizerConfig | None = None
     """Optimizer configuration to use. If not set, Adam will be used."""
 
+    reinit_tokens: list[str] | bool = False
+    """
+    if set, will reinitialize the embeddings for the given tokens. If True, will reinitialize the default tokens
+    for llama3's tokenizer
```

Comment:
Maybe 1 line of motivation for when you want to do this?

Reply:
Removed this, needs more experimenting - sorry this PR was messy at first!